### PR TITLE
Makefiles: Respect DMD/DIFFABLE variables for man pages

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
Makefile invocation:
```make -C ../dmd/docs DMD=/dev/shm/dtest/work/repo/dlang.org/.generated/stable_dmd-2.088.0/dmd2/linux/bin64/dmd DIFFABLE=1 build```

Actually used dmd:
```/home/dtest/DAutoTest/work/build/bin/dmd```

### CODE REVIEW

**Comment by MoonlightSentinel** [comment]
`DIFFABLE` was lost in translation as well

**Review by marler8997 — APPROVED** [review]

File: docs/Makefile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 DMD=dmd
 GENERATED=../generated
 DMD_MAN_PAGE=man/man1/dmd.1
+DIFFABLE=1
```
Comment (CyberShadow) [inline docs/Makefile]: Don't you want `DIFFABLE` to be off by default (i.e. normal documentation builds include timestamps, commit SHA1s, etc.)?

Comment (MoonlightSentinel) [reply docs/Makefile]: Indeed

Comment (MoonlightSentinel) [reply docs/Makefile]: Indeed

**Review by wilzbach — APPROVED** [review]

**Pull request merged**